Tools/Scripts/libraries/webkitcorepy/webkitcorepy/string_utils.py

```diff
@@ -92,23 +92,31 @@ def out_of(number, base):
     return '[{}{}/{}]'.format(' ' * (len(base) - len(number)), number, base)
 
 
-def elapsed(seconds):
-    if seconds <= 0:
-        return 'no time'
-    elif seconds < 1:
-        return 'less than a second'
-
-    seconds = int(round(seconds))
+def elapsed(seconds: float, precision: int = None, shorthand: bool = False):
```

Comment:
Can we make the new arguments keyword only arguments? i.e.:

```suggestion
def elapsed(seconds: float, *, precision: int = None, shorthand: bool = False):
```

Also many of the test cases violate the type specified for `seconds`; we probably want either `Union[int, float]` or `numbers.Real`.